--- a/configure.in
+++ b/configure.in
@@ -664,13 +664,6 @@
   \
   Makefile \
   \
-  mockpp/docs/Makefile \
-  mockpp/docs/api/Makefile \
-  mockpp/docs/en/Makefile \
-  mockpp/docs/en/common/Makefile \
-  mockpp/docs/en/images/Makefile \
-  mockpp/docs/en/images/callouts/Makefile \
-  \
   mockpp/po/Makefile \
   mockpp/config/Makefile \
   mockpp/tests/Makefile \
--- a/mockpp/Makefile.am
+++ b/mockpp/Makefile.am
@@ -3,7 +3,7 @@
 
 SUBDIRS = chaining visiting compat util builder constraint matcher stub \
           production framework \
-          . config docs tests po examples
+          . config tests po examples
 
 lib_LTLIBRARIES = libmockpp.la
 
